CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ endif ()
 set(ENABLE_MORE_WARNINGS OFF)
 
 # Set C specific flags
-set(CMAKE_C_FLAGS "-D_GNU_SOURCE -std=gnu99 -fPIC ${CMAKE_C_FLAGS}")
+set(CMAKE_C_FLAGS "-D_GNU_SOURCE -std=gnu11 -fPIC ${CMAKE_C_FLAGS}")
```

Comment:
Note: that this will update the C requirements for Celix from gnu99 to gnu11. 

I have no issue with this, but maybe it good to get some broader feedback on this.
@rlenferink , @rbulter , @ErjanAltena and @stegemr Do any of you have any uses updating the Celix C standard from gnu99 to gnu11? . Of I we have no issue, can we also introduce the use of `#pragma once` for C code?

Reply:
I'd better follow pubsub_admin's usage of gcc built-in atomic operations  to avoid introducing C11.